external/source/exploits/CVE-2019-1458/dllmain.cpp

```diff
@@ -0,0 +1,75 @@

```

Comment:
I couldn't find a way to pass an argument (containing the payload) to the reflective loader, so I had to fall back to the old method (for the sandbox escape, but not for the standalone module).

Reply:
Unfortunately it's still required for the sandbox escape. In the local exploit we can inject the payload into the process memory:
https://github.com/rapid7/metasploit-framework/pull/13817/files#diff-9fe6c7df53f3453217fcff2b8d27b29aR125
In the chrome exploit there isn't an easy way to do this so the `PAYLOAD:` string is replaced: https://github.com/rapid7/metasploit-framework/pull/13817/files#diff-591453026163506af84eb6534ccb31b1R82

Reply:
Gotcha, thanks for clarifying.